### ISSUE TITLE
Default export from next-routes

### ISSUE THREAD

**Issue by dbenfouzari** [fridays/next-routes#181]
Hi !

I'm trying to get next-routes working with TypeScript.

I have this code right now :
```ts
import Routes from 'next-routes';

const routes = new Routes();

routes
  .add('about')
  .add('post', '/p/:id')

export default routes;
```

But when I'm trying to compile I get this error : 

> const routes = new Routes();
                             ^
TypeError: next_routes_1.default is not a constructor

I followed example [here](https://github.com/fridays/next-routes/blob/master/typings/tests/basic.ts)

Could you please help me ?

**Comment by prevostc** [comment]
Same here. Solved it with: 
```
import * as createRoutes from "next-routes"
import Routes from "next-routes"

// @ts-ignore Types are broken
const routes: Routes = createRoutes()

routes
  .add("about")

export default routes
export const Link = routes.Link
export const Router = routes.Router
```

**Comment by prevostc** [comment]
It works but requires a `@ts-ignore` and has pretty a bad semantic. I would like to see this fixed too :) @fridays do you accept PR on this issue? Is there something we do not know?

**Comment by dbenfouzari** [comment]
@prevostc I agree, that's not good and should be fixed but.. It works atleast :D

**Comment by smkhalsa** [comment]
@fridays I believe there's a pending PR to fix this. Can we get it merged please?

**Comment by atombender** [comment]
Anything happening with this?
